components/inputs/input-time.js

```diff
@@ -168,12 +168,16 @@ class InputTime extends LitElement {
 						aria-controls="${this._dropdownId}"
 						aria-labelledby="${this._dropdownId}-label"
 						@change="${this._handleChange}"
+						@keydown="${this._handleKeydown}"
```

Comment:
What was the motivation to switch from keypress -> keydown? We try to avoid keydown if we can since folks can hit it accidentally, whereas keypress only fires after a down and then an up.

Reply:
I used `keydown` to use the same events as `input-date` and because when I went to look up the difference between the two, I found `keypress` is apparently deprecated https://developer.mozilla.org/en-US/docs/Web/API/Document/keypress_event - it's not technically a switch because apparently that `keypress` listener didn't do anything (see other conversation)

Reply:
Crazy, that's my new learned thing for today.